pysc2/lib/renderer_human.py

```diff
@@ -676,6 +677,25 @@ def draw_overlay(self, surf):
     surf.surf.blit(text, (3, 3))
 
   @sw.decorate
+  def draw_shield_energy_status(self, surf):
+    """Draw the energy and shield status under the unit."""
+    for u, p in self._visible_units():
+      if self._camera.intersects_circle(p, u.radius):
+        if u.alliance != 3:
```

Comment:
Why is checking that the unit isn't neutral needed? If it is actually needed, could you at least use the name instead of number?